::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2017 Mirantis Inc., Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Fri, 8 Dec 2017 20:30:46 +0100
Subject: [PATCH] salt-master-init.sh: Apply OPNFV Fuel patches

Some of Fuel@OPNFV patches need to be applied before the reclass
storage.node state is ran for Salt Master, i.e. between installing
salt-formula-* packages and configuring the Salt Master salt services.

JIRA: FUEL-310

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 salt-master-init.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/salt-master-init.sh b/salt-master-init.sh
index 343324c..a4ec138 100755
--- a/salt-master-init.sh
+++ b/salt-master-init.sh
@@ -284,6 +284,7 @@ options
     system_config

     saltmaster_bootstrap &&\
+    /root/fuel/mcp/patches/patch.sh /root/fuel/mcp/patches/patches_init.list formulas &&\
     saltmaster_init &&\

     verify_salt_minions
